package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
this file should not be modified by you

Reply:
We want to start targeting `1.0.1-{pre-release}` correct?